.gitignore

```diff
@@ -80,6 +80,7 @@ maps
 deployment/static
 deployment/backups
 deployment/media
+deployment/docker-osm-healthcare/settings
```

Comment:
do we really want to ignore the settings folder? Dont you have your style file in there?

Reply:
ah its because you are using a submodule?